### PR TITLE
add vrrp_garp_master_refresh / vrrp_garp_lower_prio_delay params

### PR DESCRIPTION
**Pull request by a-dekker**
#### This Pull Request (PR) fixes the following issues
Fixes https://github.com/voxpupuli/puppet-keepalived/issues/205

- All spec tests pass.
- Beaker tests do provide the expected keepalived config file.

### CODE REVIEW

File: manifests/global_defs.pp [hunk 1]
```diff
@@ -66,6 +72,8 @@
   $enable_traps                                   = undef,
   Optional[Boolean] $vrrp_higher_prio_send_advert = undef,
   Optional[Integer] $vrrp_garp_lower_prio_repeat  = undef,
+  Optional[Integer] $vrrp_garp_master_refresh     = undef,
```
Comment (bastelfreak) [inline manifests/global_defs.pp]: do you know which integer values are the minimum and maximum? If so, we can enforce them as well.

**Comment by bastelfreak** [comment]
Hi @a-dekker, thanks for the PR. Can you please take a look at the used email address in the commit? It isn't associated with your github account.

File: manifests/global_defs.pp (hunk 1 above)
Comment (a-dekker) [reply manifests/global_defs.pp]: It's not mentioned in the documentation, only that the default is 0 (no refreshing). 
Looking at https://github.com/acassen/keepalived/blob/245e2ac65dade95f5ccbecb4b1d9d478966418af/keepalived/core/global_parser.c#L680 it is set to UINT_MAX, but I am no C expert.

**Comment by a-dekker** [comment]
The email will be set to my work mail address and not my personal mail address.

**Pull request merged**